cli/cmd/sync_v3.go

```diff
@@ -633,6 +633,17 @@ func syncConnectionV3(ctx context.Context, syncOptions syncV3Options) (syncErr e
 		}, durationsPerTable)
 	}
 
+	for i := range destinationsClients {
+		if destinationSpecs[i].WriteMode == specs.WriteModeOverwriteDeleteStale {
+			// Table names might have changed due to transformers
+			updatedTablesForDeleteStale := tableNameChanger.UpdateTableNames(destinationSpecs[i].Name, tablesForDeleteStale)
+			if err := deleteStale(writeClients[i], updatedTablesForDeleteStale, sourceName, syncTime); err != nil {
+				return err
+			}
+		}
+	}
+
+	syncDurationMs := time.Since(syncTime)
```

Comment:
We should probably not re-calculate this to avoid `syncTimeTook = time.Since(syncTime)` having a slightly different value in the logs vs sync summary